webpack.config.js

```diff
@@ -2,20 +2,17 @@
 const merge = require('webpack-merge');
 const fs = require('fs');
 const path = require('path');
-const dotenv = require('dotenv');
+const getEnv = require('./src/server/env');
 
 const appDirectory = fs.realpathSync(process.cwd());
 const resolvePath = relativePath => path.resolve(appDirectory, relativePath);
 
-// Load in environment variables configured in `.env` file. Be aware the `.env` committed is changed by bake-scripts when on real servers.
-const result = dotenv.config();
-
-if (result.error) {
-  throw result.error;
-}
+// Load in environment variables configured in `.env` files.
+const DOT_ENV_CONFIG = getEnv();
 
 // `shell` parameter populated via CLI, e.g. --env.platform=web
 module.exports = (shell = {}) => {
+  const APP_ENV = process.env.APP_ENV || 'live';
```

Comment:
By having this default to live doesn't that make the logic to fall back to `.env` if `process.env.APP_ENV` is not set obsolete? 

EG: https://github.com/bbc/simorgh/pull/1204/files#diff-3891f3f26878f8e17bd04f1a222621bbR7

Reply:
This is just a safeguard incase https://github.com/bbc/simorgh/blob/WebpackEnviromentMultiBuild/.env#L5 doesnt exist for some reason